Support Freeze Tenant via API

SCFireblocks opened the pull request:
Support Freeze Tenant via API

amimagid committed:
changed return type to pending instead of TokenLink (#188)

* changed return type to pending instead of TokenLink

* fixed request object

* Update src/fireblocks-sdk.ts

Co-authored-by: Idan Yael <idanya@users.noreply.github.com>

---------

Co-authored-by: Ami Magid <amagid@fireblocks.com>
Co-authored-by: Idan Yael <idanya@users.noreply.github.com>

amiryadid-fb committed:
Add NCW types

amper-fb committed:
add NCW feature (#192)

* feat: NCW SDK

amiryadid-fb committed:
clean types

amper-fb committed:
fix: add NCW methods (#197)

Commit:
release 5.5.0

yuval-fireblocks committed:
rename base url

yuval-fireblocks committed:
rename var

tomervil committed:
Merge pull request #198 from fireblocks/rename-ncw-endpoint

Rename NCW base path

Commit:
release 5.5.1

zoharsf reviewed (commented):
I failed to find any changes related to the Freeze Tenant feature in the entire MR

amimagid committed:
added registry endpoint for uploading contracts (#190)

* added registry endpoint for uploading contracts

* added all interfaces

* Update src/fireblocks-sdk.ts

Co-authored-by: Idan Yael <idanya@users.noreply.github.com>

* forgot import

---------

Co-authored-by: Ami Magid <amagid@fireblocks.com>
Co-authored-by: Idan Yael <idanya@users.noreply.github.com>

Commit:
release 5.6.0

barakfireblocks committed:
added getAddressesWithPaging

barakfireblocks committed:
added AddressesWithPagingResponse

dpetrovic-fb committed:
Add Smart Transfers user groups endpoints (#202)

amiryadid-fb committed:
chainIds is optional

amiryadid-fb committed:
Merge branch 'master' into wallet-link-ncw-support

aviba committed:
Merge pull request #193 from fireblocks/wallet-link-ncw-support

barakfireblocks committed:
added AddressesWithPagingResponse

lazars14-f committed:
feat: added list owned assets and ncw for NFTs

lazars14-f committed:
refactor: TokenWithBalanceBase

aviba committed:
Merge pull request #206 from fireblocks/add-ncw-for-nfts

Added list owned assets endpoint and NCW for NFTs

Commit:
release 5.6.1

gadiguy committed:
added bulk functions

gadiguy committed:
removed redundant files

asafs932 committed:
feat: adding tap endpoints (#207)

vsiskin committed:
Change Collection and Token response (#205)

* add list owned assets endpoint, add status filter to list owned collections

* remove duplicated field from NFTOwnedCollectionsFilter

* remove duplicated interface NFTOwnedAssetsFilter

rade-fireblocks committed:
Travel Rule Configuration and Policy routes (#187)

* Feat: Travel Rule Configuration and Policy routes

* Exposing external Travel Rule endpoints

View TR screening and post/screening policy

Bypass screening during service outages.

Inbound transaction delay

Outbound transaction delay

* Update path and method naming

* Update paths remove unecessary braces

* Unify Update Screening Policy configuration endpoints add interfaces

* Rename paramethar for Screening Policy configuration

* Update JS doc for updateTravelRulePolicyConfiguration parameter description

* Fix update screening policy update URL

* Add undescore on Update url

* remove duplicate import

Commit:
release 5.7.0

SCFireblocks committed:
Merge pull request #185 from fireblocks/get-transactions-ascending

added sort to transaction filter

or109 committed:
contract-registry - adding endpoints (#210)

* ContractTemplateDto

* ContractTemplateDto

* tokenization endpoints

* LeanContractTemplateDto

* getLinkedTokens using pageSize and pageCursor

* sourcecode and bytecode

* Web3PagedResponse

* remove bytecode and constructorEncodedParams

* constructorParameters optional

* Unlink a token from the tenant

---------

Co-authored-by: orman <omanzur@fireblocks.com>

Commit:
release 5.8.0

Commit:
Merge remote-tracking branch 'origin/master' into staking-sdk

# Conflicts:
#	src/fireblocks-sdk.ts

Commit:
add staking sdk endpoints

amper-fb committed:
feat: NCW supported assets (#215)

lazars14-f committed:
feat: added multi nft token status update endpoint (#214)

AmitShe committed:
COR-31331: Add support paged exchange account endpoint (#213)

Commit:
release 5.9.0

Commit:
add staking sdk endpoints

Commit:
Merge remote-tracking branch 'origin/master' into staking-sdk

# Conflicts:
#	src/fireblocks-sdk.ts

Commit:
fix getChainInfo

Commit:
separate sdk

barakfireblocks committed:
getPaginatedAddresses

barakfireblocks committed:
Merge branch 'master' into COR-33565

barakfireblocks committed:
.

gadiguy committed:
fixed comments reviews

barakfireblocks committed:
.

barakfireblocks committed:
.

barakfireblocks committed:
.

barakfireblocks committed:
fix bug

yahav-fireblocks committed:
staking-sdk: integrate staking APIs for use with FireblocksSDK as required

gadiguy committed:
no need for async

gadiguy committed:
Merge branch 'master' into FEAT-COR-23078-bulk-methods

gadiguy committed:
fixed merge error

yahav-fireblocks committed:
staking-sdk: mr comments implemented, add execute action request dto for staking/unstake/withdraw

yahav-fireblocks committed:
staking-sdk: executeStakingAction type fix

barakfireblocks committed:
cr fix

aviba committed:
Merge pull request #216 from fireblocks/staking-sdk

Staking sdk

gadiguy committed:
removed duplicates

zoharsf committed:
Merge pull request #209 from gadiguy/FEAT-COR-23078-bulk-methods

added bulk functions

Tamirsh507 committed:
Merge branch 'master' into get-asset-wallets

barakfireblocks committed:
.

barakfireblocks committed:
Merge branch 'master' into COR-33565

barakfireblocks committed:
cr fix

or109 committed:
contract-service - adding endpoints  (#221)

* adding new endpoints

* longDescription

* omit vendorId

* responses

* PendingTokenLinkDto

* remove TokenLinkPermissionEntry

---------

Co-authored-by: orman <omanzur@fireblocks.com>

barakfireblocks committed:
cr fix

Commit:
release 5.10.0

yahav-fireblocks committed:
staking-sdk: update staking sdk

ranmitel committed:
Merge pull request #223 from fireblocks/Add-getPaginatedAddresses-method

Add getPaginatedAddresses method

gadiguy committed:
Revert mistake in updateVaultAccount

zoharsf committed:
Merge pull request #224 from gadiguy/patch-1

Revert mistake in updateVaultAccount

aviba committed:
Merge pull request #222 from fireblocks/staking-sdk

staking-sdk: update staking sdk

Commit:
release 5.11.0

amiryadid-fb committed:
Add PeerType

Tamirsh507 committed:
Merge pull request #150 from fireblocks/get-asset-wallets

Add get asset wallets paginated endpoint

yanlevi committed:
Added sourceWalletId for NCW under TransactionFilter (#228)

Added relevant query argument for NCW filtering

Commit:
release 5.12.0

dependabot[bot] committed:
Bump semver from 5.7.1 to 5.7.2 (#182)

Bumps [semver](https://github.com/npm/node-semver) from 5.7.1 to 5.7.2.
- [Release notes](https://github.com/npm/node-semver/releases)
- [Changelog](https://github.com/npm/node-semver/blob/v5.7.2/CHANGELOG.md)
- [Commits](https://github.com/npm/node-semver/compare/v5.7.1...v5.7.2)

---
updated-dependencies:
- dependency-name: semver
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

amper-fb committed:
feat: ncw latest backup (#225)

dependabot[bot] committed:
Bump axios from 0.27.2 to 1.6.0 (#231)

* Bump axios from 0.27.2 to 1.6.0

Bumps [axios](https://github.com/axios/axios) from 0.27.2 to 1.6.0.
- [Release notes](https://github.com/axios/axios/releases)
- [Changelog](https://github.com/axios/axios/blob/v1.x/CHANGELOG.md)
- [Commits](https://github.com/axios/axios/compare/v0.27.2...v1.6.0)

---
updated-dependencies:
- dependency-name: axios
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

* update typescript & @notabene/pii-sdk dependencies

---------

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Yoav Bar-Zeev <ybarzeev@fireblocks.com>

Commit:
release 5.13.0

GrishaDev committed:
User management (#189)

* sdk

* fix type

* update

* users type

* reset-device

* change sdk routes

* management subroute

* cr

* fix tslint

---------

Co-authored-by: gregory <gregory@fireblocks.com>

rade-fireblocks committed:
Exposing external aml endpoints for screening setting (#226)

* Exposing External Endpoints - AML Policy and Configuration
* View AML screening / post screening policy
* Bypass screening during service outages, inbound and outband transaction delay
* Expose endpoint for Screening setting
* View AML screening / post screening policy
* Remove logs from PII enc
* Add proper response types for screening endpoints
* Rename Screening Transaction Status

AlonFireblocks committed:
Fix empty querystring scenario (#235)

* fix
* syntax
* semantics
* Align JSDoc

---------

Co-authored-by: YoavBZ <yoav.bz4@gmail.com>

Commit:
release 5.14.0

aviba committed:
Merge pull request #227 from fireblocks/there-is-no-way-to-create-a-transaction-with-our-sdk-to-peer-type-contract

feat: Add PeerType

nitaiaharoni1 committed:
Support tenant's OTA configuration functionality (#237)

OTA Configurations

Commit:
release 5.15.0

GrishaDev committed:
Rename management paths (#238)

Co-authored-by: Gregory Koltoun <gregory@fireblocks.com>

Commit:
release 5.15.1

yuval-fireblocks committed:
fix: add missing params (#241)

Commit:
release 5.16.0

dean-ab committed:
Audit logs api (#240)

* COR-29935: Updated audits API

* COR-29935: Added jsdoc param

* COR-29935: Paginated audit logs

Commit:
release 5.17.0

dpetrovic-fb committed:
add missing amountUsd (#239)

amper-fb committed:
feat: Add request interceptor and httpsAgent options (#245)

* feat: optional httpsAgent

* add interceptor

Commit:
release 5.18.0

ankitshubham97 committed:
update type of addressFormat to string | undefined (#243)

or109 committed:
Tokenization updated API (#247)

* Get contract ABI function by contractId

* types.ts

* tokenization updated API

* refId

* docs

* add interfaces

* getContractTemplates

* Contract Templates

* tokenMetadata

* TokenLink

---------

Co-authored-by: orman <omanzur@fireblocks.com>

Commit:
release 5.18.1

dean-ab committed:
Renamed management routes (#248)

vsiskin committed:
Add spam changes to js sdk (#250)

* add spam filter and updateTokenOwnesrhipSpamStatus

* change function name

* add spam and status response properties

* fix imports

* BaseTokenWithBalance status should be required

* fix typo

invalid-email-address committed:
release 5.19.0

orenyomtov committed:
Add `errorDescription` to `TransactionResponse`

amimagid committed:
Merge pull request #251 from fireblocks/COR-47703-contract-errors

Add `errorDescription` to `TransactionResponse`

invalid-email-address committed:
release 5.20.0

yahav-fireblocks committed:
staking-sdk: update staking sdk

yahav-fireblocks committed:
staking-sdk: fix deleted import

or109 committed:
assetId (#256)

Co-authored-by: orman <omanzur@fireblocks.com>

yahavamar committed:
Merge pull request #254 from fireblocks/staking-sdk

staking-sdk: update staking sdk

yahav-fireblocks committed:
staking-sdk: fix PositionStatus missing statuses, StakeResponse response property and getPosition return type

yahav-fireblocks committed:
staking-sdk: mr comments

dsahni committed:
Update README.md (#261)

yahavamar committed:
Merge pull request #260 from fireblocks/staking-sdk

staking-sdk: fix PositionStatus missing statuses, StakeResponse respo…

invalid-email-address committed:
release 5.21.0

vsiskin committed:
add createNewAsset function

rade-fireblocks committed:
Add supported screening assets route (#259)

* Add supporte screening assets route

* Remove unused import

* Remove an empty line from ScreeningSupportedProviders enum

vsiskin committed:
update docs

vsiskin committed:
add response interface, change function name

vsiskin committed:
use "" instead of ''

vsiskin committed:
Merge branch 'master' into add-create-new-asset-support

# Conflicts:
#	src/fireblocks-sdk.ts

rade-fireblocks committed:
Remove colon from supported screening URL (#265)

* Remove colon from supported asset url

invalid-email-address committed:
release 5.22.0

or109 committed:
Tokenization - Add displayName (#262)

* displayName

* displayName

* functionValue is optional

---------

Co-authored-by: orman <omanzur@fireblocks.com>

invalid-email-address committed:
release 5.23.0

yuval-fireblocks committed:
add wallet\device setup state EPs

yuval-fireblocks committed:
fix: lint and desc.

yuval-fireblocks committed:
Merge pull request #267 from fireblocks/ncw-device-setup

NCW: wallet & device setup status EPs

goweiss committed:
COR-55933 - Add algorithm id to asset response

vsiskin committed:
update RegisterAssetResponse

vsiskin committed:
update RegisterAssetResponse, id -> legacyId

vsiskin committed:
update RegisterAssetResponse

aviba committed:
Merge pull request #264 from fireblocks/add-create-new-asset-support

Add registerNewAsset function

invalid-email-address committed:
release 5.24.0

yuval-fireblocks committed:
add unspent input ep

yuval-fireblocks committed:
fix type and response

yuval-fireblocks committed:
Merge remote-tracking branch 'origin/master' into ncw-unspent-inputs

yuval-fireblocks committed:
Merge pull request #272 from fireblocks/ncw-unspent-inputs

NCW: get unspent inputs

invalid-email-address committed:
release 5.25.0

or109 committed:
Tokenization api changes (#273)

* /tokenization/templates

* update /tokenization/contracts endpoints

* update /contract_interactions endpoints

* update getContractsByFilter endpoint

* contractTemplateId

* pathWithParams

* contractTemplateId

* SupportedBlockchain

* baseAssetId

---------

Co-authored-by: orman <omanzur@fireblocks.com>

invalid-email-address committed:
release 5.25.1

phanigovindaraju committed:
Update Tokenization DTOs and deprecate unused route (#276)

* Remove constructor endpoint

* Remove constructorParams

* Change EVMTokenCreateParamsDto

* Update parameter name

* Adding back constructor params to support integration tests

* Removing constructor params

yuval-fireblocks committed:
logic

yuval-fireblocks committed:
fix path

yuval-fireblocks committed:
minor

invalid-email-address committed:
release 5.26.0

yuval-fireblocks committed:
fix: type

amimagid committed:
added getContractByAddress function

amimagid committed:
added getContractByAddress function

amimagid committed:
Merge pull request #279 from fireblocks/fetch-address-by-chain-and-tx-hash

Fetch address by chain and tx hash

AlonFireblocks committed:
Merge pull request #271 from goweiss/COR-55933-Add-algorithm-id-to-asset-response

Add algorithm id to asset response

invalid-email-address committed:
release 5.27.0

MMaaliFireblocks committed:
functions instead of abi

amimagid committed:
Merge pull request #280 from brohamgoham/fix-wrong-path-contract-interactions

functions instead of abi

invalid-email-address committed:
release 5.27.1

yuval-fireblocks committed:
Merge remote-tracking branch 'origin/master' into ncw-set-wallet-required-algos

AlonFireblocks committed:
Merge pull request #278 from fireblocks/fix-get-wallet-devices

fix: getWalletDevices type

AlonFireblocks committed:
Merge pull request #281 from fireblocks/ncw-set-wallet-required-algos

feat: set ncw wallet required algorithms

invalid-email-address committed:
release 5.27.2

barakfireblocks committed:
Add-rescan-txs

barakfireblocks committed:
format

or109 committed:
Add Tokenization collections API endpoints

or109 committed:
Add Tokenization collections API endpoints

or109 committed:
mintCollectionToken

or109 committed:
BurnCollectionTokenRequest

or109 committed:
BurnCollectionTokenRequest

or109 committed:
change

or109 committed:
change

or109 committed:
mint burn

or109 committed:
LinkedCollectionMetadataDto

or109 committed:
Mint NFT

amimagid committed:
Merge pull request #285 from fireblocks/add-collections-api

Add Tokenization Collections API

invalid-email-address committed:
release 5.28.0

trongvnau committed:
[OC-4662] Tokenization Collection API - Changed token metadata objects' fields from camel Case to snake case (#288)

* changed collection api token metadata fields to snake_case

* format

* format

---------

Co-authored-by: Trong Nguyen <tnguyen@fireblocks.com>

invalid-email-address committed:
release 5.28.1

trongvnau committed:
removed unnecessary field from BurnCollectionTokenRequest (#289)

Co-authored-by: Trong Nguyen <tnguyen@fireblocks.com>

phanigovindaraju committed:
Add link contract routes (#287)

* Remove constructor endpoint

* Remove constructorParams

* Change EVMTokenCreateParamsDto

* Update parameter name

* Adding back constructor params to support integration tests

* Removing constructor params

* Add new routes

* Fix the descriptions

* Remove blank lines

* add params

* Fix endpoint

* Address comment

* Fix types

* Address comment

vsiskin committed:
add set asset price api (#277)

* add set asset price api

* change param types for setAssetPrice endpoint

---------

Co-authored-by: Lazar Stijakovic <lstijakovic@fireblocks.com>

invalid-email-address committed:
release 5.29.0

barakfireblocks committed:
rescanTxs

barakfireblocks committed:
Merge branch 'master' into Add-rescan-txs

barakfireblocks committed:
beta

barakfireblocks committed:
Merge pull request #283 from fireblocks/Add-rescan-txs

Add-rescan-txs

phanigovindaraju committed:
Fix issue with add abi (#291)

* Fix issue with add abi

* Naming change

* Fix route

yuval-fireblocks committed:
add maxTotalFee tx-request param (#290)

* add maxTxFee

* rename to maxTotalFee

YoavBZ committed:
Update publish-npm.yml

YoavBZ committed:
Update publish-npm.yml

invalid-email-address committed:
release 5.31.2

trongvnau committed:
[OC-5077] Updated fetch-abi to fetch_abi (#293)

* updated fetch-abi to fetch_abi

* format

---------

Co-authored-by: Trong Nguyen <tnguyen@fireblocks.com>

skfb committed:
define get/set methods for vasp to vault assign

skfb committed:
inline documentation

skfb committed:
inline documentation, precise response

skfb committed:
remove null type from vaspDid return value

skfb committed:
inline documentation, precise request

skfb committed:
v1 prefix to endpoints

mlyberopoulos committed:
OC-5460 - Add externalId in WriteCallFunctionDto && OC-5461 - Add SDK to support fee and feeLevel (#298)

* Update fee and feeLevel

* Add externalId

* Update ContractDeployRequest with fee

* Update types.ts

* Update types.ts

skfb committed:
Merge pull request #296 from fireblocks/SPT-2123-get/set-vasp-assigned-to-vault

Get/set VASP assigned to vault

invalid-email-address committed:
release 5.32.0

yuval-fireblocks committed:
change ethNetwork from number to string

yuval-fireblocks committed:
Merge pull request #299 from fireblocks/fix-asset-response

[ncw] change ethNetwork from number to string on asset response

m-awais-ahmad committed:
add useGasless option (#300)

* add useGasless option

* add fee option to ReadCallFunctionDto

dolevmiz1 committed:
fix: package.json & package-lock.json to reduce vulnerabilities (#304)

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-ELLIPTIC-7577916
- https://snyk.io/vuln/SNYK-JS-ELLIPTIC-7577917
- https://snyk.io/vuln/SNYK-JS-ELLIPTIC-7577918
- https://snyk.io/vuln/SNYK-JS-ELLIPTIC-8187303
- https://snyk.io/vuln/SNYK-JS-ELLIPTIC-8172694

Co-authored-by: snyk-bot <snyk-bot@snyk.io>

dolevmiz1 committed:
fix: upgrade jsonwebtoken from 9.0.0 to 9.0.2 (#303)

Snyk has created this PR to upgrade jsonwebtoken from 9.0.0 to 9.0.2.

See this package in npm:
jsonwebtoken

See this project in Snyk:
https://app.snyk.io/org/ori-fvw/project/7ca60b8f-454a-4430-ab7a-b46193792580?utm_source=github&utm_medium=referral&page=upgrade-pr

Co-authored-by: snyk-bot <snyk-bot@snyk.io>

dolevmiz1 committed:
fix: upgrade axios from 1.6.0 to 1.7.7 (#301)

Snyk has created this PR to upgrade axios from 1.6.0 to 1.7.7.

See this package in npm:
axios

See this project in Snyk:
https://app.snyk.io/org/ori-fvw/project/7ca60b8f-454a-4430-ab7a-b46193792580?utm_source=github&utm_medium=referral&page=upgrade-pr

Co-authored-by: snyk-bot <snyk-bot@snyk.io>
Co-authored-by: Yoav Bar-Zeev <ybarzeev@fireblocks.com>

invalid-email-address committed:
release 5.33.0

m-awais-ahmad committed:
support fallback function in abi (#306)

goweiss committed:
GetPublicKey For NCW (#294)

* WLT-658 - GetPublicKey For NCW

added two new functions for NCW GetPublicKey

* WLT-658 - GetPublicKey For NCW

code review fixes

* WLT-658 - GetPublicKey For NCW

* WLT-658 - GetPublicKey For NCW

yahav-fireblocks committed:
staking-sdk: add solana split support

yahavamar committed:
Merge pull request #312 from fireblocks/staking-sdk

staking-sdk: add solana split support

goweiss committed:
WLT-1945 - [Fireblocks JS SDK] Fix missing field in return type of WalletSetupStatusResponse (#313)

invalid-email-address committed:
release 5.34.0

yahav-fireblocks committed:
staking-sdk: add amount to the UnstakeRequestDto for liquid unstake

chenshoo committed:
Merge pull request #316 from fireblocks/staking-sdk

staking-sdk: add amount to the UnstakeRequestDto for liquid unstake

yuval-fireblocks committed:
[ncw] delete signing algo EP (#309)

* add EP

* fix url

* fix api method

* typo

yuval-fireblocks committed:
NCW: add getWalletDevice method to retrieve specific wallet device details (#315)

* add getWalletDevice method to retrieve specific wallet device details

* deviceId param to doc

* doc

lazars14-f committed:
feat: added asset and blockchain beta apis

lazars14-f committed:
lint: spacing

lazars14-f committed:
fix: removed idempotency key

SlavaSereb committed:
Merge pull request #317 from lazars14-f/master

Add asset and blockchain beta apis

invalid-email-address committed:
release 5.35.0

m-awais-ahmad committed:
feat: add getTransactionReceipt method and TransactionReceiptResponseDto type (#318)

rade-fireblocks committed:
Travel Rule - beneficiaryVASPdid is optional

SlavaSereb committed:
Merge pull request #320 from fireblocks/SPT-2669-travel-rule-fireblocks-legacy-js-sdk-beneficiary-vas-pdid-is-optional

Travel Rule - beneficiaryVASPdid is optional

invalid-email-address committed:
release 5.36.0

m-awais-ahmad committed:
mak type non optional (#321)

MotyRose committed:
feat: add getInternalWalletAssets method method for paginated asset retrieval

MotyRose committed:
Merge pull request #322 from fireblocks/WLT-2588-Unmanaged-wallet-paginated-endpoint

Add paginated endpoint to get balance on /internal_wallets/<wallet id>/assets

lazars14-f committed:
feat: change asset and blockchain apis response (#319)

* feat: changed asset and blockchain apis response

* fix: separate scope for register asset

lazars14-f committed:
feat: blockchains and assets endpoints GA (#326)

* feat: changed asset and blockchain apis response

* fix: separate scope for register asset

* feat: blockchain and asset listing endpoints GA

* feat: changed names for interfaces for response

invalid-email-address committed:
release 5.37.0

lazars14-f committed:
Add ids filter for assets and blockchains listing endpoints (#327)

* feat: changed asset and blockchain apis response

* fix: separate scope for register asset

* feat: blockchain and asset listing endpoints GA

* feat: changed names for interfaces for response

* feat: added ids filter for listing assets and blockchains endpoints

itzukerman committed:
added ncw header

itzukerman committed:
remived if

itzukerman committed:
fix

itzukerman committed:
fix

itzukerman committed:
rename

itzukerman committed:
rename

itzukerman committed:
mr fix

YoavBZ committed:
Update GitHub actions packages

itzukerman committed:
Merge branch 'master' into WLT-3822-add-ncw-header

itzukerman committed:
Merge pull request #329 from fireblocks/WLT-3822-add-ncw-header

added ncw header

invalid-email-address committed:
release 5.38.0

vsiskin committed:
Add updateAssetUserMetadata endpoint (#330)

* add updateAssetUserMetadata endpoint

* send expected payload as a param

lazars14-f committed:
feat: Add verified to asset response dto (#331)

* feat: changed asset and blockchain apis response

* fix: separate scope for register asset

* feat: blockchain and asset listing endpoints GA

* feat: changed names for interfaces for response

* feat: added ids filter for listing assets and blockchains endpoints

* feat: added verified to list asset response

vsiskin committed:
add asset features support (#334)

asafs932 committed:
fix: Update axios package version (#336)

asafs932 committed:
fix: Internal Infra Change To Publish With Github App (#337)

invalid-email-address committed:
release 5.39.0

asafs932 committed:
fix: Internal Infra Change To Publish With OIDC (#339)

invalid-email-address committed:
release 5.39.0

asafs932 committed:
fix: Update Axios Version (#343)

invalid-email-address committed:
release 5.39.1
